File: 151004/Bordilovsky/DC_REST/Entities/Label.cs
using System.ComponentModel.DataAnnotations;

namespace DC_REST.Entities
{
	public class Label
	{
		public int Id { get; set; }

		public string Name { get; set; }

		//public List<Issue> Issues { get; set; } = new();
		public List<Issue_Label> Issue_Labels { get; set; } = new();
	}
}

File: 151004/Terebey/NoSql/publisher/src/main/java/org/education/bean/Label.java
package org.education.bean;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@NoArgsConstructor
@Setter
@Getter
@Entity(name = "tbl_label")
public class Label {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int id;
    String name;

    @ManyToMany
    @JoinTable(
            name = "IssueLabel",
            joinColumns = @JoinColumn(name = "issueId", referencedColumnName = "id"),
            inverseJoinColumns = @JoinColumn(name = "labelId", referencedColumnName = "id")
    )
    List<Issue> issues;
}